definitions/features/dynflow_sidekiq.rb

```diff
@@ -8,14 +8,9 @@ class Features::DynflowSidekiq < ForemanMaintain::Feature
   end
 
   def config_files
-    # Workaround until foreman-installer can deploy scaled workers
-    service_symlinks = configured_instances.map do |service|
-      "/etc/systemd/system/multi-user.target.wants/#{service}.service"
-    end
     [
       '/etc/foreman/dynflow',
```

Comment:
To what degree should we be backing up config files vs. the installer putting them in place for us?

Reply:
That's a very good question, I have no answer for